source/likely.js

```diff
@@ -4,6 +4,7 @@ var likely = require('./index.js');
 var history = require('./history');
 
 window.addEventListener('load', function () {
+    likely.defaults = likely.getDefaultOptions();
```

Comment:
Like, I mean, we definitely need some default options, but there’s no need even in `getDefaultOptions()`.